www_src/pages/discover/discover.less

```diff
@@ -1,3 +1,46 @@
 #discover {
+  display: block;
+  position: relative;
 
+  width: 100%;
+  height: 100%;
+  padding: 5% 0;
+
+  .card {
+    display: block;
+    position: relative;
+
+    width: 90%;
+    margin: 0 auto 5% auto;
+    
+    color: #666;
+    text-decoration: none; 
+    background-color: white;
+    box-shadow: 1px 2px 2px 2px rgba(0, 0, 0, 0.1);
+
+    .thumbnail {
+      padding: 8px 0 0 0;
+
+      img {
+        display: block;
+        position: relative;
+
+        width: 97%;
+        margin: 0 auto;
+        background: #efefef;
+      }
+    }
+
+    .meta {
+      width: 97%;
```

Comment:
why 97? (mostly asking because anything that isn't 50 or 100 is probably a "this can be make perfect with `calc`" instance)